### DISCUSSION TITLE
Why the >gv mapping for indent in visual mode?

### DISCUSSION THREAD

**Discussion by bluss** [LazyVim/LazyVim#1187] [General]
I found this in the lazyvim config:

```lua
-- better indenting
map("v", "<", "<gv")
map("v", ">", ">gv")
```

And for me it broke my habits from Vim in general. I haven't discovered how these keymaps improve indenting - if somebody knows, can you explain the intention?

This is what I can do *without* these mappings, which is useful:

1. Shift-V and select three lines downwards visually
2. `>` to indent them once
3. Repeat indent of the same three lines using `.`

The mapping breaks the third step in two ways - the cursor position changes (without mapping: cursor at top of visual block, with mapping: bottom of block) and it's still in visual mode where I can't use . for some reason.

**Comment by folke** [comment]
If you don't like the mapping, you can easily remove it in your `lua/config/keymaps.lua`

**Reply by bluss** [reply]
Yes, no problem, that's the key to solving this for me. The tricky part maybe was to find where the mapping was set, and the remaining question is why is it better, do you want to explain how you use it? :slightly_smiling_face:

**Reply by folke** [reply]
because I like repeating the indent with the same key?

**Reply by bluss** [reply]
I think I just missed the obvious way to use it! (blinded by my own habits). It was just not obvious to me how the "unexpected" binding worked, sorry about that. :slightly_smiling_face:  Bothering you not intended, posted here in discussions so that anyone can answer.
